tools/transformer/src/transformations/allOf.js

```diff
@@ -28,7 +29,9 @@ function applyAllOfTransformations(api) {
   );
 
   for (let { path } of allOfTransformations) {
-    transformAllOf(path, api);
+    if(!isInlineModel(path)) {
```

Comment:
Ignore inline models for `allOf` transformation as the parent will never have properties or be referenced by the children.